ports/admin_handler.go

```diff
@@ -125,6 +125,12 @@ func (s *AdminHandler) AddBuilder(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(http.StatusBadRequest)
 		return
 	}
+	if builder.Network == "" {
+		s.log.Error("Failed to unmarshal request body", "err", err)
+		w.WriteHeader(http.StatusBadRequest)
```

Comment:
should we start returning error bodies? check out these helpers: https://github.com/flashbots/system-api/blob/9cb82ae11286526a7dc20493251332f1f5e38874/systemapi/server.go#L294-L312